### PR TITLE
[FP8 tests] Enable several fp8 tests

### PR DESCRIPTION
**Pull request by Devjiu**
This commit enables several fp8 tests, that uses `tl.range`. Requires PR #40. 

Complete the following tasks before sending your PR, and replace `[ ]` with
`[x]` to indicate you have done them.

- [x] I am not making a trivial change, such as fixing a typo in a comment.
- [x] I have written a PR description following these
  [rules](https://cbea.ms/git-commit/#why-not-how).
- [x] I have run `pre-commit run --from-ref origin/main --to-ref HEAD`.
- [x] This PR does not need a test because `FILL THIS IN`.
- [x] I have not added any `lit` tests.

### CODE REVIEW

File: third_party/cpu/lib/TritonCPUTransforms/ConvertUnsupportedOps.cpp [hunk 1]
```diff
@@ -167,6 +167,34 @@ struct ConvertBf16Abs : public OpRewritePattern<math::AbsFOp> {
...
+
+    Location loc = op.getLoc();
+    Value src = op.getOperand();
+    Value intSrc =
+        rewriter.create<arith::BitcastOp>(loc, toInt8(op.getType()), src);
+
+    // Mask out the sign bit
+    auto num_bits = getElementTypeOrSelf(op.getType()).getIntOrFloatBitWidth();
+    assert(num_bits <= 16);
+    auto mask = (1u << (num_bits - 1u)) - 1u;
```
Comment (ienkovich) [inline third_party/cpu/lib/TritonCPUTransforms/ConvertUnsupportedOps.cpp:186]: This var is unused.

File: third_party/cpu/lib/TritonCPUTransforms/ConvertUnsupportedOps.cpp [hunk 2]
```diff
@@ -167,6 +167,34 @@ struct ConvertBf16Abs : public OpRewritePattern<math::AbsFOp> {
...
+    TypedAttr maskAttr = rewriter.getI8IntegerAttr(0x7f);
+    if (auto vecTy = dyn_cast<VectorType>(intSrc.getType()))
+      maskAttr = SplatElementsAttr::get(vecTy, maskAttr);
```
Comment (ienkovich) [inline third_party/cpu/lib/TritonCPUTransforms/ConvertUnsupportedOps.cpp:189]: You can use `cst_like` helper from `OptCommon.h` here.

**Review by minjang — APPROVED** [review]
LGTM

**Comment by Devjiu** [comment]
@minjang  we have an issue with tests, looks like they are too long and tests are cancelled due to timeout.

**Pull request merged**